msc/static/js/AddressCopy.js

```diff
@@ -0,0 +1,19 @@
+function FillBilling(f) {
```

Comment:
We didn't discuss this before but probably a good practice to make any of the JS files that correspond to a specific view, the same name as the view.